src/InfluxDB2/WriteApi.php

```diff
@@ -2,16 +2,46 @@
 
 namespace InfluxDB2;
 
-
 use InfluxDB2\Model\WritePrecision;
 
+class PointSettings
+{
+    private $defaultTags;
+
+    public function __construct(array $defaultTags = null)
+    {
+        $this->defaultTags = is_null($defaultTags) ? [] : $defaultTags;
+    }
+
+    public function addDefaultTag(string $key, string $value)
```

Comment:
The value of tag should be evaluated "lazy".

```suggestion
    public function addDefaultTag(string $key, string $expression)
```